### PR TITLE
Tag 1.0

### PR DESCRIPTION
**Pull request by DanieleAsnicar**
The project has achieved a satisfactory initial degree of maturity and functionality. Therefore, it can be merged into main and tagged.

Implemented features:
- Options screen
- Game screen
- Resizable game board
- Scrollable game board
- Start/Pause button
- Clear game board button
- Selectable game options
- Fading visual effects

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ Communication with the backend from QML to C++ and vice-versa is always and excl
 - Now that the FSM receives the `backend` by reference, should the FSM be directly responsible of carrying out back-end actions such as reinitialising the game board? Or is it better that the FSM just informs the `Backend` that the `Backend` has to perform a certain action, and the `Backend` actually carries out that action?
 - The automatic stop is probably inefficient, because it checks for equality between the `m_currentMatrix` and the `m_nextMatrix` at every game loop. Should a timer be used to rarify the checks?
 - Add the option to enable / disable the cell animation, and to set the percentage with respect to the game timer period.
+- Consider if GameStateMachine and other classes should be singletons
```
Comment (GammaSigma1234) [inline README.md:72]: Consider adding already discussed TODO: using an asymmetrical fading effect (i.e. only a fade-in, not a fade-out).

**Review by GammaSigma1234 — CHANGES_REQUESTED** [review]
Please edit the TODO list as discussed.

File: README.md (hunk 1 above)
Comment (DanieleAsnicar) [reply README.md:72]: Added discussed TODO item and amended commit.

Comment (GammaSigma1234) [reply README.md:72]: OK

**Review by GammaSigma1234 — APPROVED** [review]

**Pull request merged**